### PR TITLE
Documenting Near/power/small fuzzify raster algs

### PR DESCRIPTION
**Pull request by rosaguilar**
Goal: Adds the documentation of the last three fuzzify algs. This PR, also fixes an error in the image of the linear formula (previous PR). 

Ticket(s): #4390
- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst [hunk 1]
```diff
@@ -1622,4 +1935,10 @@ Outputs
 .. |fuzzy_large_formula| image:: img/fuzzy_large_formula.png
    :height: 3.2em
 .. |fuzzy_linear_formula| image:: img/fuzzy_linear_formula.png
+   :height: 3.8em
+.. |near_formula| image:: img/fuzzy_near_formula.png
+   :height: 2.5em   
+.. |power_formula| image:: img/fuzzy_power_formula.png
+   :height: 4.4em   
+.. |small_formula| image:: img/fuzzy_small_formula.png
```
Comment (havatv) [inline docs/user_manual/processing_algs/qgis/rasteranalysis.rst:1943]: Both this and the following ones look a bit large for inline usage, but I guess you have checked, and found them OK.

Comment (rosaguilar) [reply docs/user_manual/processing_algs/qgis/rasteranalysis.rst:1943]: I checked both. I was hesitating with the biggest (power) but I think is ok. See the pics below:
image

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst [hunk 2]
```diff
@@ -308,6 +314,313 @@ Outputs
...
+     - Description
+   * - **Fuzzified raster**
+     - ``OUTPUT``
+     - [same as input]
+     - Output raster layer containing the result
+   * - **CRS authority identifier**
+     - ``CRS_AUTHID``
+     - [crs]
+     - The coordinate reference system of the output raster layer
+   * - **CRS authority identifier**
```
[the remark this answers is not in the document]
Comment (rosaguilar) [reply docs/user_manual/processing_algs/qgis/rasteranalysis.rst]: Thanks for noticing, it is fixed.

**Comment by havatv** [comment]
Thanks, @rosaguilar !

**Pull request merged**